src/createWidgetBase.ts

```diff
@@ -160,11 +151,8 @@ const createWidget: WidgetBaseFactory = createEvented
 				return properties;
 			},
 
-			classes: [],
-
 			getNode(this: Widget<WidgetProperties>): DNode {
-				const tag = formatTagNameAndClasses(this.tagName, this.classes);
-				return v(tag, this.getNodeAttributes(), this.getChildrenNodes());
+				return v('div', {}, this.getChildrenNodes());
```

Comment:
My working assumption is that the dataviz mixin would just wrap any dataviz component with an `svg` node (essentially doing the same as what the mixin does now by setting the tagName to `svg`)